app/views/questionnaire.py

```diff
@@ -672,7 +672,8 @@ def _get_front_end_navigation(answer_store, current_location, metadata, routing_
     completed_blocks = get_completed_blocks(current_user)
     navigation = Navigation(g.schema, answer_store, metadata, completed_blocks, routing_path)
     block_json = g.schema.get_block(current_location.block_id)
-    if block_json is not None and block_json['type'] in ('Question', 'Interstitial', 'Confirmation', 'Summary'):
+    if block_json['type'] in ('Question', 'Interstitial', 'Confirmation', 'SectionSummary', 'Summary') and \
```

Comment:
Just change this to `if block_json['type'] != 'Introduction' and block_json is not None:`

Reply:
or even try `if block_json['type'] != 'Introduction':` as i'm not sure why `block_json` would ever be `None`